ui/locales/app.en.json

```
@@ -190,7 +190,7 @@
   "lyr_voxel_geneva_logk_label": "Geneva: Hydraulic conductivity",
   "lyr_voxel_visp_litho_label": "Visp: Modelling Unit",
   "lyr_voxel_visp_logk_label": "Visp: Hydraulic conductivity",
-  "lyr_voxel_rheintal_klasse_label": "Rheintal - Entire model (fine)",
+  "lyr_voxel_rheintal_klasse_label": "St.Galler Rheintal - Permanently ",
```

Ich glaube diese Übersetzung stimmt nicht:
```suggestion
  "lyr_voxel_rheintal_klasse_label": "St.Galler Rheintal - Permeability",
```